src/pages/Models/Details/ModelDetails.test.js

```diff
@@ -36,7 +36,7 @@ describe("ModelDetail Container", () => {
     const wrapper = mount(
       <Provider store={store}>
         <MemoryRouter
-          initialEntries={["/models/space-man@external/frontend-ci"]}
+          initialEntries={["/models/spaceman@external/hadoopspark"]}
```

Comment:
The previous model didn't even exist...This test only passed because it was simply checking for the existence of the tables not that they were populated.